server/routes/api/index.js

```diff
@@ -0,0 +1,8 @@
+const router = require('express').Router();
+
+router.use('/challenges', require('./challenges'));
```

Comment:
Again - why are we inlineing requiring modules?

Reply:
Fixed